server/auth/types/authentication_type.ts

```diff
@@ -118,7 +119,7 @@ export abstract class AuthenticationType implements IAuthenticationType {
         const additonalAuthHeader = this.getAdditionalAuthHeader(request);
         Object.assign(authHeaders, additonalAuthHeader);
         authInfo = await this.securityClient.authinfo(request, additonalAuthHeader);
-        cookie = await this.getCookie(request, authInfo);
+        cookie = this.getCookie(request, authInfo);
```

Comment:
Why removing `await`?

Reply:
`getCookie` is no longer an async function. So remove the await.